k-distribution/include/builtin/domains.k

```diff
@@ -155,6 +155,23 @@ module MAP
   syntax KItem ::= choice(Map)                      [function, hook(MAP.choice), klabel(Map:choice)]
 endmodule
 
+module MAP-SYMBOLIC [symbolic]
+  imports MAP
+  imports K-EQUAL
+
+  rule ((K1 |-> V1) MAP) [ K2 ] => V1         requires K1  ==K K2
+  rule ((K1 |-> V1) MAP) [ K2 ] => MAP [ K2 ] requires K1 =/=K K2
+
+  rule ([ K1 <- K2 ] MAP) [ K2 ] => V1         requires K1  ==K K2
+  rule ([ K1 <- K2 ] MAP) [ K2 ] => MAP [ K2 ] requires K1 =/=K K2
```

Comment:
syntax? `[ K1 <- K2 ] MAP` instead of `MAP [ K1 <- K2 ]`?

Reply:
fixed